auditbeat/Jenkinsfile.yml

```diff
@@ -41,6 +41,9 @@ stages:
     crosscompile:
         make: "make -C auditbeat crosscompile"
         stage: mandatory
+    macosActions:
+        githubAction: "mage build unitTest"
+        stage: extended
```

Comment:
For the time being, let's use the mandatory stage to run earlier, since the `extended` stage runs after all the `mandatory` stages have finished successfully.

```suggestion
    macosActions:
        githubAction: "mage build unitTest"
        stage: mandatory
```

Reply:
Added this to the PR.